bindings/ros/README.md

```diff
@@ -0,0 +1,40 @@
+# ROS Wrapper for the ADI ToF library
+
+## Overview
+This ROS package facilitates depth and IR data acquisition and processing for the Analog Devices depth cameras.
+
+## Installation
+
+- **Install the recommended [ROS distribution](http://wiki.ros.org/Distributions) for your operating system**
```

Comment:
do these steps work for windows too? Or is there something different that the user needs to do if he/she is on Windows?

Reply:
This ROS package does not offer support for Windows yet.